content/kit/webcam.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Webcam
+---
+
+To see the outside world, a webcam is used to capture images for them to be processed. The webcam in the kit is a [TeckNet C016](http://www.tecknet.co.uk/c016.html).
+
+![The Webcam](http://www.tecknet.co.uk/media/catalog/product/cache/1/thumbnail/9df78eab33525d08d6e5fb8d27136e95/c/0/c016_webcam_4_.jpg?width=30pc)
```

Comment:
I'd recommend we host the image ourselves, since it's possible that TeckNet might reorganise their website or discontinue the product sometime in the near future.

Reply:
Sounds like a good idea. I think there's a couple other places we hotlink images. I'd suggest we make the change to internalising images in a separate PR

Reply:
Sounds good.